tests/char_volume_test.cpp

```diff
@@ -44,19 +44,18 @@ TEST_CASE( "character_baseline_volumes", "[volume]" )
     Character &you = get_player_character();
     REQUIRE( you.get_mutations().empty() );
     REQUIRE( you.height() == 175 );
-    CHECK( you.get_base_volume() == 70_liter );
```

Comment:
There should still be a check for baseline volume to make sure it doesn't get too wild. If need be, you can force certain values (e.g. the body fat %), as that would still provide useful coverage over *the rest of the values*.

Reply:
Good point. I'll make some tests to make sure height, weight, and volume is within reasonable bounds.